[Bug] edit profile error

**Description**

[ERROR] :경광등: 예외 발생 in CurrentUserDetail
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/rest_framework/views.py", line 512, in dispatch
    response = handler(request, *args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/rest_framework/generics.py", line 262, in patch
    return self.partial_update(request, *args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/rest_framework/mixins.py", line 82, in partial_update
    return self.update(request, *args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/rest_framework/mixins.py", line 68, in update
    self.perform_update(serializer)
  File "/usr/local/lib/python3.9/contextlib.py", line 79, in inner
    return func(*args, **kwds)
  File "/app/adoorback/account/views.py", line 877, in perform_update
    notis_to_change.update(
NameError: name 'notis_to_change' is not defined

--- a/adoorback/account/views.py
+++ b/adoorback/account/views.py
@@ -12,6 +12,7 @@ from django.contrib.auth import authenticate, logout
 from django.core import exceptions
 from django.core.exceptions import ObjectDoesNotExist
 from django.contrib.auth import get_user_model
+from django.contrib.contenttypes.models import ContentType
 from django.contrib.auth.password_validation import validate_password
 from django.db import transaction, IntegrityError
 from django.db.models import Q, Case, When, Value, IntegerField, Count
@@ -874,6 +875,10 @@ class CurrentUserDetail(generics.RetrieveUpdateAPIView):
                 
                 # "became friends" notification
                 Notification = apps.get_model('notification', 'Notification')
+                user_ct = ContentType.objects.get_for_model(User)
+                notis_to_change = self.request.user.friendship_originated_notis.filter(
+                    target_type=user_ct
+                )
                 notis_to_change.update(
                     redirect_url=f"/users/{new_username}"
                 )

